tasks/main.yml

```diff
@@ -25,24 +25,26 @@
     enabled: yes
   become: yes
 
+# This is not required in all setups, but can save a task where it is needed
 - name: Install Infiniband packages
   package:
-    name: "{{ item }}"
-    state: present
-  become: yes
-  with_items:
+    name:
     - '@InfiniBand Support'
     - perftest
     - infiniband-diags
+    state: present
+  become: yes
+  when: gluster_cluster_transport_mode == "rdma"
 
 - name: Install Gluster packages
   package:
-    name: "{{ item }}"
-    state: present
+    name:
     - glusterfs-server
     - glusterfs-client
     - glusterfs-rdma
```

Comment:
Is gluster-rdma required when the transport mode is not selected?

Reply:
Shouldn't be: https://gluster-documentations.readthedocs.io/en/latest/Install-Guide/Install/